### PR TITLE
Update to latest Flask and PyLTI 1.3

### PR DESCRIPTION
**Pull request by bagofarms**
This branch updates all of the packages to work with the latest version of Flask. Be aware that it uses a Docker image that is only compatible with arm64 platforms. I have not finished testing it, so I would like some help with that. Currently it runs, but the LTI launch is not completing properly.

### CODE REVIEW

**Comment by bagofarms** [comment]
Ok, it's working now, but it shows `Your browser prohibits saving cookies in an iframe.`. If I click the provided link to open in a new tab, it correctly shows my user's email address.

**Review by Thetwam — APPROVED** [review]
At a glance, looks good. I'll defer final judgement to the team of TRs that've been using this as an example.

File: requirements.txt [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-Flask-Migrate==3.1.0
-PyLTI1p3==1.8.4
-cryptography==3.2
-mysqlclient==2.0.3
-Werkzeug==1.0.1
-pycrypto
\ No newline at end of file
+boto3==1.26.*
+botocore==1.29.*
+canvasapi==3.0.0
```
Comment (Thetwam) [inline requirements.txt]: technically LTI isn't exclusive to Canvas. Is there a specific reason for canvasapi's inclusion?

Comment (bagofarms) [reply requirements.txt]: I must have left that in from the experiments I was running. I removed it.

**Pull request merged**